tests/full/test.fmf

```diff
@@ -0,0 +1,21 @@
+summary: Run the full tmt test suite in an external system
+test: ./test.sh
+framework: beakerlib
+adjust:
+- when: distro != fedora
+  require+:
+  - library(epel/epel)
+require:
+- rsync
+- git-core
+- rpm-build
+- make
+- procps-ng
+- python3-docutils
```

Comment:
python3-docutils can't be in require - on RHEL-like systems codeready-builder repo (and its alternatives) needs to be enabled first by the test itself